cmd/gravy/simulate.go

```diff
@@ -43,16 +45,17 @@ func GetDataStores(dailywindowFilename string) map[string]db.Store {
 }
 
 // Writes a CSV header.
-func WriteCSVHeader(headers []string, out *bufio.Writer) {
+func WriteCSVHeader(headers []string, out *os.File) {
```

Comment:
What was the reason for switching to `os.File` instead of the bufio?

Reply:
Well, os.File already has a WriteString function, so it's not much worse. The ultimate reason was I needed to switch to a file so I can Close it in another function. (We could add a WriteCloser if we like.)